graphql/image_resolver.go

```diff
@@ -0,0 +1,34 @@
+package graphql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen/model/distro"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/evergreen/thirdparty"
+)
+
+// Distros is the resolver for the distros field.
+func (r *imageResolver) Distros(ctx context.Context, obj *thirdparty.Image) ([]*model.APIDistro, error) {
+	if obj.ImageID == "" {
```

Comment:
let's add a nil check like these resolvers so we don't panic 

https://github.com/bynn/evergreen/blob/ee98c0f5b0aa68d8af270f2d1c15474869b5694c/graphql/distro_resolver.go#L198

Reply:
fixed!